src/main.tsx

```diff
@@ -1,10 +1,10 @@
-import { StrictMode } from 'react'
-import { createRoot } from 'react-dom/client'
-import App from './App.tsx'
-import './index.css'
+import React from "react";
+import ReactDOM from "react-dom/client";
+import App from "./App";
+import "./index.css";
```

Comment:
Probably not an issue right now, but a reminder for us to set upp eslint and prettier configs for the project, so our PR:s don't keep switching between `"` and `'` 😅 (or some other setting where our personal eslint-configs differ)